Components/button.css

```diff
@@ -3,7 +3,7 @@
     background: var(--default-color);
     color: var(--background-color);
     width: max-content;
-    padding: 1rem 1rem ;
+    padding: 1rem 1rem;
```

Comment:
Use global variables for padding, margin, and gaps.

Reply:
Okay